oxd-server/src/main/java/org/gluu/oxd/server/persistence/SqlPersistenceServiceImpl.java

```diff
@@ -23,10 +25,12 @@
     private static final Logger LOG = LoggerFactory.getLogger(SqlPersistenceServiceImpl.class);
 
     private SqlPersistenceProvider provider;
+    private ConfigurationService configurationService;
 
     @Inject
-    public SqlPersistenceServiceImpl(SqlPersistenceProvider provider) {
+    public SqlPersistenceServiceImpl(SqlPersistenceProvider provider, ConfigurationService configurationService) {
         this.provider = provider;
+        this.configurationService = configurationService;
```

Comment:
`this.expiredObjectColumnName = getKeyColumnName(configurationService);`
Then we calculate it one time during `SqlPersistenceServiceImpl` creation and then re-use in all other places.

Reply:
done